### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by BO41**
This is a automated code cleanup performed by Android Studio. Follow up from TeamNewPipe/NewPipeExtractor#100

I hope those changes don't break anything. They shouldn't and I always run the gradle unit tests.
(just like with the NewPipeExtractor)

replaces  #1625

### CODE REVIEW

**Comment by BO41** [comment]
I hope I did resolve the merge conflicts correctly.
Always took the version of dev

If not I will redo everything but using dev and not master this time

File: app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java [hunk 1]
```diff
@@ -316,16 +326,18 @@ private void updateSubscription(final ChannelInfo info) {
...
                 if (DEBUG) Log.d(TAG, "Changed subscription status to this channel!");
         };
 
-        final Consumer<Throwable> onError = (@NonNull Throwable throwable) ->
-                onUnrecoverableError(throwable,
-                        UserAction.SUBSCRIPTION,
-                        NewPipe.getNameOfService(currentInfo.getServiceId()),
-                        "Subscription Change",
-                        R.string.subscription_change_failed);
+        final Consumer<Throwable> onError = new Consumer<Throwable>() {
```
Comment (theScrabi) [inline app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java]: I wonder why this lambda is replaced by a definition again.

File: app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.java [hunk 2]
```diff
@@ -232,12 +232,13 @@ public void onNext(SubscriptionEntity subscriptionEntity) {
                 if (!itemsLoaded.contains(subscriptionEntity.getServiceId() + subscriptionEntity.getUrl())) {
                     subscriptionService.getChannelInfo(subscriptionEntity)
                             .observeOn(AndroidSchedulers.mainThread())
-                            .onErrorComplete(
-                                    (@io.reactivex.annotations.NonNull Throwable throwable) ->
-                                            FeedFragment.super.onError(throwable))
-                            .subscribe(
-                                    getChannelInfoObserver(subscriptionEntity.getServiceId(),
-                                            subscriptionEntity.getUrl()));
+                            .onErrorComplete(new Predicate<Throwable>() {
```
Comment (theScrabi) [inline app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.java]: Here the same.

File: app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java [hunk 3]
```diff
@@ -276,38 +275,49 @@ private void monitorSubscription(final ChannelInfo info) {
...
 
-        final Consumer<Throwable> onError = (@NonNull Throwable throwable) ->
-                onUnrecoverableError(throwable,
-                        UserAction.SUBSCRIPTION,
-                        NewPipe.getNameOfService(info.getServiceId()),
-                        "Updating Subscription for " + info.getUrl(),
-                        R.string.subscription_update_failed);
+        final Consumer<Throwable> onError = new Consumer<Throwable>() {
+            @Override
+            public void accept(@NonNull Throwable throwable) throws Exception {
```
Comment (theScrabi) [inline app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java]: I wonder why this lambda is replaced by a definition again.

File: app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java [hunk 4]
```diff
@@ -276,38 +275,49 @@ private void monitorSubscription(final ChannelInfo info) {
...
+                return o;
+            }
         };
     }
 
     private Function<Object, Object> mapOnUnsubscribe(final SubscriptionEntity subscription) {
-        return (@NonNull Object o) -> {
-            subscriptionService.subscriptionTable().delete(subscription);
-            return o;
+        return new Function<Object, Object>() {
```
Comment (theScrabi) [inline app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java]: Do not remove lambda.

File: app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java [hunk 5]
```diff
@@ -276,38 +275,49 @@ private void monitorSubscription(final ChannelInfo info) {
...
 
     }
 
     private Function<Object, Object> mapOnSubscribe(final SubscriptionEntity subscription) {
-        return (@NonNull Object o) -> {
-            subscriptionService.subscriptionTable().insert(subscription);
-            return o;
+        return new Function<Object, Object>() {
+            @Override
+            public Object apply(@NonNull Object o) throws Exception {
```
Comment (theScrabi) [inline app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java]: Do not remove lambda.

File: app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java [hunk 6]
```diff
@@ -276,38 +275,49 @@ private void monitorSubscription(final ChannelInfo info) {
...
+            }
         };
     }
 
     private void updateSubscription(final ChannelInfo info) {
         if (DEBUG) Log.d(TAG, "updateSubscription() called with: info = [" + info + "]");
-        final Action onComplete = () -> {
+        final Action onComplete = new Action() {
+            @Override
+            public void run() throws Exception {
```
Comment (theScrabi) [inline app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java]: Do not remove lambda.

File: app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java [hunk 7]
```diff
@@ -276,38 +275,49 @@ private void monitorSubscription(final ChannelInfo info) {
                 // so only update the UI for the latest emission ("sync" the subscribe button's state)
                 .debounce(100, TimeUnit.MILLISECONDS)
                 .observeOn(AndroidSchedulers.mainThread())
-                .subscribe((List<SubscriptionEntity> subscriptionEntities) ->
-                        updateSubscribeButton(!subscriptionEntities.isEmpty())
-                        , onError));
+                .subscribe(new Consumer<List<SubscriptionEntity>>() {
+                    @Override
+                    public void accept(List<SubscriptionEntity> subscriptionEntities) throws Exception {
```
Comment (theScrabi) [inline app/src/main/java/org/schabi/newpipe/fragments/list/channel/ChannelFragment.java]: Do not remove lambda.

File: app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.java (hunk 2 above)
Comment (theScrabi) [reply app/src/main/java/org/schabi/newpipe/local/feed/FeedFragment.java]: Do not remove lambda.

File: app/src/main/java/org/schabi/newpipe/App.java [hunk 8]
```diff
@@ -162,11 +162,10 @@ private void reportException(@NonNull final Throwable throwable) {
         });
     }
 
-    private ImageLoaderConfiguration getImageLoaderConfigurations(final int memoryCacheSizeMb,
-                                                                  final int diskCacheSizeMb) {
+    private ImageLoaderConfiguration getImageLoaderConfigurations(final int memoryCacheSizeMb) {
         return new ImageLoaderConfiguration.Builder(this)
                 .memoryCache(new LRULimitedMemoryCache(memoryCacheSizeMb * 1024 * 1024))
-                .diskCacheSize(diskCacheSizeMb * 1024 * 1024)
+                .diskCacheSize(50 * 1024 * 1024)
```
Comment (theScrabi) [inline app/src/main/java/org/schabi/newpipe/App.java]: Do not use inline `50` use `diskCacheSizeMb`.

**Comment by BO41** [comment]
forgot to test after the rebase, sorry.

okay I think it would be best if I redo everything, this time at the dev branch and while paying more attention

**Comment by BO41** [comment]
@theScrabi just fyi, you can create a "Inspection profile" for the profile. Android Studio will use it then.

**Comment by theScrabi** [comment]
Would be a good idea. :)

**Pull request closed without merging**